ablytest/sandbox.go

```
@@ -117,39 +142,54 @@ func NewRealtime(opts ...ably.ClientOption) (*Sandbox, *ably.Realtime) {
 
-func NewSandboxWithEndpoint(config *Config, endpoint string) (*Sandbox, error) {
+var sharedApp = sync.OnceValues(func() (*Sandbox, error) {
+	return provisionSandbox(Endpoint)
+})
+
+// CloseSharedApp deletes the shared app if it was provisioned. It is intended to
+// be called once from TestMain after all tests have run.
+func CloseSharedApp() error {
+	app, err := sharedApp()
+	if err != nil {
+		return nil // never provisioned successfully; nothing to delete
```

golangci-lint (2.12.2)

[error] error is not nil (line 179) but it returns nil

(nilerr)